test/unit_Polytope.jl

```diff
@@ -350,6 +350,22 @@ if test_suite_polyhedra
         twoB = 2.0*B
         @test X ⊆ twoB && twoB ⊆ X
 
+        # concrete minkowski difference (special case, where A == A ⊖ B ⊕ B)
+        A2 = BallInf(N[0, 0], N(5))
+        B2 = BallInf(N[0, 0], N(2))
+        C2 = minkowski_difference(A2,B2)
+        @test C2 ⊆ BallInf(N[0, 0], N(3)) && BallInf(N[0, 0], N(3)) ⊆ C2
+
+        # concrete minkowski difference for unbounded P (HPolyhedra)
+        mx1 = 2.0
```

Comment:
It also works with other number types (for `Rational{Int}` I had to use `Ball1`):
```julia
        mx1 = N(2)
        mx2 = N(5)
        P3 = HPolyhedron(N[mx1 0; 0 mx2], N[3, 3])
        radius = 2
        Q3 = Ball1(N[0, 0], N(radius))
        C3 = minkowski_difference(P3, Q3)
        C3_res = HPolyhedron(N[mx1 0; 0 mx2], N[3 - mx1*radius, 3 - mx2*radius])
        @test C3 ⊆ C3_res && C3_res ⊆ C3
```

Reply:
Should I just replace the unit test with this code?

Reply:
Yes, that was the idea :+1:

Reply:
great, we can merge after that change 👍